apps/twap/src/app/orderbook/settlement.ts

```diff
@@ -0,0 +1,45 @@
+import { Column, PrimaryColumn, Entity } from 'typeorm';
+import {
+  bigIntToString,
+  bufferToString,
+  stringToBigInt,
+  stringToBuffer,
+} from '../utils/transformers';
+
+@Entity({ name: 'settlements' })
+export class Settlement {
+  @PrimaryColumn('bigint', {
+    name: 'block_number',
+    transformer: { from: stringToBigInt, to: bigIntToString },
+  })
+  blockNumber: bigint;
+
+  @PrimaryColumn('bigint', {
+    name: 'log_index',
+    transformer: { from: stringToBigInt, to: bigIntToString },
+  })
+  logIndex: bigint;
+
+  @Column('bytea', {
```

Comment:
Should we add a `name` prop here also?

Reply:
Also what is this field actually?
Name of the solver?

Reply:
Address of the solver, from what I can see in the DB.

We can't really adjust the data type for these, since they are what backend team has in place. Entities under orderbook table exist to give us a way to read the DB itself, essentially, so it just defines what is in place.